cluster/charts/rook-ceph/values.yaml

```diff
@@ -87,6 +87,9 @@ csi:
   # sidecar with CSI provisioner pod, to enable set it to true.
   enableOMAPGenerator: false
 
+  # Set replicas for csi provisioner deployment.
+  # provisionerReplicas: 2
```

Comment:
How about uncommenting this? We don't expect the default to ever change (unlike the csi version settings that we comment out), so I think it is more readable if uncommented.